ceph-salt-formula/salt/ceph-salt/apply/tuned-off.sls

```
@@ -0,0 +1,34 @@
+
+{% import 'macros.yml' as macros %}
+
+{% if 'latency' not in grains['ceph-salt']['roles'] and 'throughput' not in grains['ceph-salt']['roles'] %}
+
+{{ macros.begin_stage('Disabling tuned') }}
+
+stop tuned:
+  service.dead:
+    - name: tuned
+    - enable: False
+    - failhard: true
+
+/etc/tuned/ceph-latency/:
+  file.absent
```

Missing "failhard":
```
remove ceph-latency:
  file.absent:
    - name: /etc/tuned/ceph-latency/
    - failhard: True
```

---

You want to fail on missing files?